controllers/servers/messages.py

```diff
@@ -67,7 +67,8 @@
     " detected host definition {} in Ready phase"
 CREATING_NEW_HOST_DEFINITION = 'Creating host Definition: {}'
 VERIFY_HOST_IS_UNDEFINED = 'Verifying that host {} is undefined from {} secret in {} namespace'
-CREATE_EVENT_FOR_HOST_DEFINITION = 'Creating event for host definition: {} error event: {}'
+CREATE_ERROR_EVENT_FOR_HOST_DEFINITION = 'Creating event for host definition: {} error event: {}'
+CREATE_SUCCESS_EVENT_FOR_HOST_DEFINITION = 'Creating event for host definition: {}'
```

Comment:
do we really need both of them?

Reply:
I am not sure what do you mean

Reply:
they look a bit similar

Reply:
what is the expected result that you want? that for success and fail events we will have the same log? or to keep it two separate logs but to put `Creating event for host definition: {}` in a common place?

Reply:
how about
```suggestion
CREATE_EVENT_FOR_HOST_DEFINITION = 'Creating event : {} for host definition: {}'
```

Reply:
what should I put in the first `{}`?

Reply:
the event

Reply:
done